### PR TITLE
feat: add projects query

### PR DESCRIPTION
**Pull request by fandujar**
This PR introduces a new flag to initialization called **google.projects.filter** which allows the exporter to query projects on a GCP Organization using labels and fields.

Example:
```
--google.projects.filter="labels.monitoring=true AND lifecycleState=ACTIVE"
```

I designed this to be a complement to the current projectID flag.
In my case, I have thousands of projects and that's why I wanted it to be flexible.

### CODE REVIEW

**Comment by fandujar** [comment]
@SuperQ Could you take a look on this PR?

File: stackdriver_exporter.go [hunk 1]
```diff
@@ -57,6 +57,10 @@ var (
 		"google.project-id", "Comma seperated list of Google Project IDs.",
 	).String()
 
+	projectsQuery = kingpin.Flag(
+		"google.projects.filter", "Google projects search query.",
```
Comment (SuperQ) [inline stackdriver_exporter.go:61]: Minor nit, let's name this consistently with the description.

```suggestion
		"google.projects-query", "Google projects search query.",
```

**Review by SuperQ — CHANGES_REQUESTED** [review]
Can you update the README with an example use of the use and a link to the Google documentation related to what's possible in the query language?

File: stackdriver_exporter.go (hunk 1 above)
Comment (fandujar) [reply stackdriver_exporter.go]: I'm gonna change all to `filter` because that's the Gcloud command line option name.

**Comment by fandujar** [comment]
@SuperQ can you review it again, please? Sorry for the delay

File: stackdriver_exporter.go [hunk 2]
```diff
@@ -57,6 +57,10 @@ var (
 		"google.project-id", "Comma seperated list of Google Project IDs.",
 	).String()
 
+	projectsQuery = kingpin.Flag(
```
Comment (SuperQ) [inline stackdriver_exporter.go:60]:
```suggestion
	projectsFilter = kingpin.Flag(
```

**Comment by netzere** [comment]
Hi @SuperQ, 
Is there a plan to release a new version so we can use this feature?

**Pull request merged**